sidebars.json

```diff
@@ -910,8 +910,47 @@
       "type": "category"
     },
     {
-      "id": "getting_started/video_pills/video_pills",
-      "type": "doc"
+      "collapsed": true,
+      "items": [
```

Comment:
I'd use the autogenerated sidebar feature to make the video adding effort easier